plugins/source/azure/codegen/recipes/sql.go

```diff
@@ -0,0 +1,326 @@
+package recipes
+
+import (
```

Comment:
This package is quite complex with many relations, some we extracted to separate files/functions and some inlined the the top level files, you can see the starting points:
https://github.com/cloudquery/cloudquery/blob/32253e9edfa677850a993a96002e59c71f95fa57/plugins/source/azure/resources/services/sql/servers.go
https://github.com/cloudquery/cloudquery/blob/32253e9edfa677850a993a96002e59c71f95fa57/plugins/source/azure/resources/services/sql/managed_instances.go

This PR makes this resource fully auto generated with each relation in a separate file